common/app/templates/inlineJS/blocking/applyRenderConditions.scala.js

```diff
@@ -101,5 +140,10 @@
     if (baseFontSize && parseInt(baseFontSize, 10) !== 16) {
         documentElement.style.fontSize = baseFontSize
     }
+
+    if (hideReaderRevenue()) {
+        docClass += ' hide-reader-revenue';
```

Comment:
@NataliaLKB sorry your comment came through after my 2nd – @joelochlann do we have a notion of somehow having paid something in js? a name for that?